src/App.js

```diff
@@ -1,13 +1,20 @@
 import React from 'react';
 import './App.scss';
 
-import todos from './api/todos';
-import users from './api/users';
+import { todos } from './api/todos';
+import { users } from './api/users';
+import TodoList from './Components/TodoList/TodoList';
```

Comment:
если в компоненте у тебя есть файл index.js, в который ты импортируешь `TodoList`, зачем в `App` снова импортировать из него?
```suggestion
import TodoList from './Components/TodoList';
```

Reply:
Воно в мене атоматично так імпортнуло